app/views/recipes/show.html.erb

```diff
@@ -31,7 +31,11 @@
     <p><%= @recipe.description %></p>
     <br>
 
-    <%= button_to "Add ingredient", new_user_recipe_recipefood_path(current_user, @recipe), method: :get %> <br>
+    <div class="recipe-show-btns">
+      <button> Generate Shopping List </button>
+      <%= button_to "Add Ingredient", new_user_recipe_recipefood_path(current_user, @recipe), method: :get %>
+    </div>
+    <br>
```

Comment:
- [Optional] I think it would be better if you made sure users are able to `sign up` through the `sign up` form, when I tried to sign up I faced the error below. I think you need to add an `input` field for users signing up to input their `name` for the form to function properly. I highly recommend you do this since it's part of the project requirements.

## Error when signing up
![Screenshot from 2023-04-20 21-43-40](https://user-images.githubusercontent.com/16392046/233458962-b84af8a9-b7b6-4c47-a179-0ee09f32b38f.png)